jenkins/opensearch-dashboards/bwc-test.jenkinsfile

```diff
@@ -0,0 +1,143 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
```

Comment:
Can you also add timeouts to both the jobs?

Reply:
Updated!